<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>971</width>
      <height>661</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>COM Setting</string>
     </attribute>
     <widget class="QComboBox" name="comboBoxPort">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnScan">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>39</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Find COM Port</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnOpen">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Open</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>AFE Data Set</string>
     </attribute>
     <widget class="QTextEdit" name="textEditTx">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>290</y>
        <width>821</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditRx">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>470</y>
        <width>821</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>126</y>
        <width>911</width>
        <height>141</height>
       </rect>
      </property>
      <property name="title">
       <string>Fixed Value Data</string>
      </property>
      <widget class="QComboBox" name="comboBoxCmd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>47</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditV1">
       <property name="geometry">
        <rect>
         <x>248</x>
         <y>48</y>
         <width>101</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSend">
       <property name="geometry">
        <rect>
         <x>701</x>
         <y>47</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditV3">
       <property name="geometry">
        <rect>
         <x>474</x>
         <y>48</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxAfeIndex">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>48</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditV2">
       <property name="geometry">
        <rect>
         <x>354</x>
         <y>48</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxPEC">
       <property name="geometry">
        <rect>
         <x>592</x>
         <y>47</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxCmdType">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>103</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxStartIndex">
       <property name="geometry">
        <rect>
         <x>152</x>
         <y>103</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxEndIndex">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>103</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditStartVolt">
       <property name="geometry">
        <rect>
         <x>356</x>
         <y>103</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditStepVolt">
       <property name="geometry">
        <rect>
         <x>475</x>
         <y>103</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSendRangeVoltage">
       <property name="geometry">
        <rect>
         <x>594</x>
         <y>103</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>14</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Group</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>86</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Group</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>155</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE Index</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE Data1(V)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE Data2(V)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>470</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE Data3(V)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>594</x>
         <y>30</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PEC</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>156</x>
         <y>85</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE Start Index</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>255</x>
         <y>85</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE End Index</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>360</x>
         <y>85</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Inital(V) </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>480</x>
         <y>85</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Data Increment(mV) </string>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>787</x>
         <y>49</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE WR Counter=0</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>911</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Parameteer</string>
      </property>
      <widget class="QComboBox" name="comboBoxTotalAFE">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>40</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnSendTotalAFE">
       <property name="geometry">
        <rect>
         <x>148</x>
         <y>40</y>
         <width>80</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>22</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>AFE Total Number</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxInitDevice">
       <property name="geometry">
        <rect>
         <x>18</x>
         <y>70</y>
         <width>171</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>All Data as Default Init(0x8000)</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnClearTx">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>410</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearRx">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>590</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>270</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Command TX:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>449</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Command RX:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Calc CRC</string>
     </attribute>
     <widget class="QLineEdit" name="lineEditCrc15Data1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc15Data2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcCrc15">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CRC15</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCalcCrc10">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>90</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CRC10</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEditCrcResult">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>591</width>
        <height>321</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Courier New</family>
       </font>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_0">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_1">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_2">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditCrc10_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClearCrcResult">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>440</y>
        <width>80</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
